tests/data/generate_data.ipynb

```diff
@@ -5,6 +5,8 @@
    "metadata": {},
```

Comment:
<p>Is this a mistake - importing pandas in a markdown cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/427/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
oops, yeah don't know how that got there